src/toolkit/res_buf.h

```diff
@@ -189,6 +227,12 @@ class ResBuf {
     return rs;
   }
 
+  /// Pops the specified number of resource objects from the buffer and returns
+  /// them as a single, squashed object.
+  typename T::Ptr PopNSquash(int n) {
+    return Squash(PopN(n));
+  }
+
```

Comment:
We discussed leaving this to the user/developer - they could just call `PopN` and take charge of squashing of they want to??

Reply:
Got it, will remove